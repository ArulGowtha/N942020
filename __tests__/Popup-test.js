import React from 'react';

import renderer from 'react-test-renderer';
import Popup from '../src/screens/Popup';

test('renders correctly', () => {
  const tree = renderer.create(<Popup />).toJSON();
  expect(tree).toMatchSnapshot();
});
