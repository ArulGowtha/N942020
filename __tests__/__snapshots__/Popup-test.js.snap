// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        Object {
          "flex": 10,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text>
          Open
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "rgba(0,0,0,.5)",
            "bottom": 0,
            "left": 0,
            "opacity": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "transform": Array [
              Object {
                "translateY": 1334,
              },
            ],
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "height": "100%",
                "justifyContent": "flex-end",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 1334,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#FFF",
                "borderTopLeftRadius": 5,
                "borderTopRightRadius": 5,
                "justifyContent": "center",
                "marginHorizontal": 10,
                "minHeight": 80,
                "transform": Array [
                  Object {
                    "translateY": 0,
                  },
                ],
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <Text>
                Close
              </Text>
            </View>
            <View
              style={
                Object {
                  "height": 300,
                  "width": 200,
                }
              }
            >
              <Text>
                Good Morning
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
